### DISCUSSION TITLE
Support for footnote/endnote notation

### DISCUSSION THREAD

**Discussion by mike3764** [vkbo/novelWriter#1013] [Ideas]
I would suggest adding support for superscripts and subscripts.  That way footnotes and endnotes could be put in a different section but could be noted in the text.  Subscripts would also allow things like chemical notation.

**Comment by vkbo** [comment]
There's already an issue on this, #342.

Adding super-/subscript in general requires adding a suitable notation for it. I'm open for suggestions.

**Reply by Ellipsine** [reply]
In Markown editors, superscript is written adding "^" before something : Myfooter^1 
For subscript, I don't know, maybe a slach or blackslash ? I would suggest something like this : Subscript_1, but the sign is already used for italic.

**Reply by vkbo** [reply]
Just going through old discussions again. Underscore in the middle of text is ignored as italics indicator, so it may work. Especially for cases where it's a simple number or character being sub/superscripted. I could implement something for short pieces of text, and add a button in the Build Tool to convert them.

Feel free to make a feature request on the sub/superscript specifically. The footnote/endnote already has a request.
